_layouts/projects.md

```diff
@@ -101,16 +101,19 @@ layout: default
             <div class="read-more"><i class="fa fa-angle-double-down fa-fw" aria-hidden="true"></i>{{ site.data.lang[lng].projects.read_more_text }}</div>
             <div class="read-less"><i class="fa fa-angle-double-up fa-fw" aria-hidden="true"></i>{{ site.data.lang[lng].projects.read_less_text }}</div>
           </a>
-        </div>
-      </div>
-      <div class="row">
-        <div class="markdown-style">
-          {{ list.post | markdownify }}
-          <a href="javascript:void(0);" class="read-more-less">
-            <i class="fa fa-angle-double-up fa-fw" aria-hidden="true"></i>{{ site.data.lang[lng].projects.read_less_text }}
-          </a>
+          <div class="markdown-style">
+            {{ list.post | markdownify }}
+          </div>
         </div>
       </div>
     </div>
   {%- endfor %}
 {%- endfor %}
+<div class="showPopUp">
```

Comment:
Confirm with client that the style of the pop up is the right one for they and show how it looks like for all resolutions.